src/components/dev-hub/input-label.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { colorMap, layer } from './theme';
+
+export const StyledLabel = styled('label')`
```

Comment:
Can we keep this consistent with the filename?
```suggestion
export const InputLabel = styled('label')`
```